server/src/main/java/edu/colostate/cs/cs414/chessirecoders/JungleServer/data/Types.java

```diff
@@ -0,0 +1,33 @@
+package edu.colostate.cs.cs414.chessirecoders.JungleServer.data;
+
+public class Types {
+
+    public enum InvitationResponseType {
+        ACCEPT,
+        REJECT
+    }
+    public enum GameOutcomeType {
+        P1_WON,
+        P2_WON,
+        DRAW
+    }
+
+    public enum PlayerColor {
+        RED,
+        BLACK
+    }
+
+    public enum GameStatus {
+        WAITING,
+        IN_PROGRESS,
+        DRAW,
+        P1_WINNER,
+        P2_WINNER
```

Comment:
Is this redundant with the GameOutcomeType? Perhaps FINISHED would suffice?

Reply:
Yeah that is redundant, which I'll fix. We still need a way to track the winner though.